### DISCUSSION TITLE
installing externals with shared dynamic libraries

### DISCUSSION THREAD

**Discussion by porres** [pierreguillot/Camomile#228] [Q&A] [answered]
Hi, we're working on a fluidsynth~ external that loads fluidsynth (https://www.fluidsynth.org/) and it needs many dependencies (fluidsynth itself, plus others that it uses). We're offering these dependencies alongside the external binary (in the same folder) for it to load in Vanilla. 

How would one go about generating a camomile version that loads this external? Is it possible at all?

cheers

**Comment by pierreguillot** [comment, marked as answer]
Hi, 
It would simplify the distribution to link the plugin with static libraries if you can do it and if there is no restriction. But if you use dynamic libraries, it should work like any other program, using relative paths and/or the default system library folder.

**Reply by pierreguillot** [reply]
@porres Is your question answered?
